3289_consteval_blocks/consteval-blocks.md

```diff
@@ -152,10 +152,10 @@ Extend the wording for plainly constant-evaluated to allow a `consteval` block [
 
 ::: std
 
-[21pre]{.pnum} A non-dependent expression or conversion is _plainly constant-evaluated_ if
+[21pre]{.pnum} An expression or conversion is _plainly constant-evaluated_ if
```

Comment:
I guess could just revert these changes then (just to avoid making future changes to this paper)